src/BuildScriptGenerator/Helpers/ListBlobsHelper.cs

```diff
@@ -2,22 +2,41 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
+using System;
 using System.Linq;
+using System.Net.Http;
 using System.Xml.Linq;
 using Microsoft.Oryx.BuildScriptGenerator.Common;
 
 namespace Microsoft.Oryx.BuildScriptGenerator
 {
     internal static class ListBlobsHelper
     {
-        public static XDocument GetAllBlobs(string sdkStorageBaseUrl, string platform, System.Net.Http.HttpClient httpClient, string oryxSdkStorageAccountAccessToken)
+        public static XDocument GetAllBlobs(string sdkStorageBaseUrl, string platform, HttpClient httpClient, string oryxSdkStorageAccountAccessToken)
         {
             var oryxSdkStorageAccountAccessArgs = oryxSdkStorageAccountAccessToken?.TrimStart(new char[] { '?' }) ?? string.Empty;
 
             var url = string.Format(SdkStorageConstants.ContainerMetadataUrlFormat, sdkStorageBaseUrl, platform, string.Empty, oryxSdkStorageAccountAccessArgs);
-            var blobList = httpClient
-                .GetStringAsync(url)
-                .Result;
+            string blobList;
+            try
+            {
+                blobList = httpClient
+                    .GetStringAsync(url)
+                    .Result;
+            }
+
```

Comment:
Nit: Not a big deal, so feel free to ignore if you feel strongly about it.
Most of the time, we don't let any spaces between the try block and the catch one, so that it's clear that all this is a common thing. Looking at the Oryx code (https://github.com/microsoft/Oryx/search?l=C%23&q=catch&type=code), putting them together seems pretty standard in our codebase.
However, again it's not a big deal so if you prefer this (and can convince most of the time to go with this new convention), I'm good with it.

Reply:
I only did it like that because otherwise stylecop fails the build because i have to have an empty line above a comment (and I cannot do an inline comment after the line either thanks to stylecop)

I don't really like it, but if you can think of a better place to put the comment, let me know!

Reply:
Honestly, I'd consider just dropping the comment? It seems pretty natural to me that if you're catching specifically this exception, this is what is thrown in the try {} block.
Alternatively, maybe putting it after the first bracket of the catch block?